app/scripts/src/selector.es.js

```diff
@@ -18,13 +18,9 @@ module.directive('languageSelector', [
             templateUrl: 'templates/language-select/language-options.html',
             scope: {},
             link: function (scope) {
-                // Make languages and current language available to template scope
                 scope.selectedLanguage = languageStorage.get();
                 scope.languageChoices = languageStorage.getLanguageChoices();
 
-                // Call this in template when changing language
-                //  @param language {object} a single language object as
-                //  returned by getLanguageChoices()
```

Comment:
Because it seemed out of place. The `changeLanguage` function doesn't take any parameters.